config/dr_cluster/manager/ramen_manager_config.yaml

```diff
@@ -3,7 +3,7 @@ kind: RamenConfig
 health:
   healthProbeBindAddress: :8081
 metrics:
-  bindAddress: 127.0.0.1:8080
+  bindAddress: 127.0.0.1:9289
```

Comment:
If we are changing the bind address here, should the proxy be given this new port as the `upstream port`?